internal/pkg/template/templates/workloads/partials/cf/publish.yml

```diff
@@ -8,33 +8,23 @@
       TopicName: !Sub '${AWS::StackName}-{{logicalIDSafe $topic.Name}}'
       KmsMasterKeyId: 'alias/aws/sns'
 
-{{- if $topic.AllowedWorkers}}
 {{logicalIDSafe $topic.Name}}SNSTopicPolicy:
-    Metadata:
-      'aws:copilot:description': 'A SNS topic policy to allow {{wordSeries $topic.AllowedWorkers "and"}} {{pluralWord (len $topic.AllowedWorkers) "service" "services"}} to consume {{$topic.Name}} events'
```

Comment:
Why do we want to remove this metadata completely?

Reply:
I opted to remove it because we're not doing anything interesting with it. Like using tags and the resource creation is v fast.